src/client/minimap.cpp

```diff
@@ -683,8 +681,14 @@ MinimapMarker *Minimap::addMarker(scene::ISceneNode *parent_node)
 
 void Minimap::removeMarker(MinimapMarker **m)
 {
-	m_markers.remove_if([ptr = *m](const auto &up) { return up.get() == ptr; });
+	MinimapMarker *ptr = *m;
 	*m = nullptr;
+
+	auto it = std::find_if(m_markers.begin(), m_markers.end(), [&] (const auto &it) {
```

Comment:
I might be a nuisance now, but I still do not see the advantage of this over `remove_if`

Reply:
And `erase_if` is only in c++20+
OK... Enough discussion on this :)